src/pages/billing/CreateNewBillingProjectWizard.test.js

```diff
@@ -128,6 +128,8 @@ describe('CreateNewBillingProjectWizard Steps', () => {
   let wizardComponent
 
   beforeEach(() => {
+    jest.resetAllMocks()
```

Comment:
example of test suite that assumes resetMocks is true.  Calling jest.resetAllMocks() keeps this test suite working as before.

Reply:
I am surprised that this is the only test that started failing when mocks are not reset by default.

Reply:
many test suites are defensively setting mocks in beforeEach, and overriding in specific tests as needed.  So they are un-affected.